hack/build-go.sh

```diff
@@ -103,7 +103,7 @@ fi
 for arg in $args; do
     if [ "${target}" = "test" ]; then
         (
-            go ${target} -v -tags "${KUBEVIRT_GO_BUILD_TAGS}" ./$arg/...
+            go ${target} -v -tags "${KUBEVIRT_GO_BUILD_TAGS}" -race ./$arg/...
```

Comment:
I only see `-race` when target isn't test: https://github.com/kubevirt/kubevirt/blob/main/hack/build-go.sh#L65

Reply:
`target` here is always test. When you run `make go-test` without any packages specified it currently runs the tests with `-race`. When you run `make WHAT=pkg/virt-launcher go-test` for example, it currently would not run with the flag. Thats why i added the flag here to ensure that the result remains the same.

Reply:
I see, thanks!